polaris.shopify.com/src/components/TokensPage/TokensPage.tsx

```diff
@@ -16,7 +16,7 @@ interface Props {
     | 'motion'
     | 'shadow'
     | 'shape'
-    | 'spacing'
+    | 'space'
```

Comment:
I'd prefer to handle this change in https://github.com/Shopify/polaris/pull/9006